package.json

```diff
@@ -11,7 +11,7 @@
     "link": "lerna link --force-local",
     "start": "npm start --prefix packages/es-components",
     "commit": "commit",
-    "postinstall": "echo '\\033[0;32mInstalling npm dependencies for sub-packages\\033[0m' && lerna bootstrap --ci",
+    "postinstall": "echo '\\033[0;32mInstalling npm dependencies for sub-packages\\033[0m' && lerna bootstrap --force-local",
```

Comment:
This should resolve issues with mismatched package versions, and lerna should enable CI in a CI environment, [according to their docs](https://github.com/lerna/lerna/tree/main/commands/bootstrap#--no-ci).